public/static/app/common/MDBQueryAdaptor.js

```diff
@@ -54,6 +54,30 @@ mciModule.factory('MDBQueryAdaptor', function() {
     }
   }
 
+  // (parser) -> (tokens) -> expr[]
+  // :returns: list of expressions, which could be compiled by compileMany
+  function asDateType(parser) {
+    return function(tokens) {
+      let expr = parser(tokens)
+      if (!expr) return [] // Bypass (no expr)
+      let date = moment.utc(expr.term)
+      if (!date.isValid()) return [] // ignore invalid dates
+      if (expr.op == '==') {
```

Comment:
For an equality operator, why not do an equality query?

Reply:
Because it isn't 'intuitive' and this was specifically asked by Jim in the ticket.
Long story short: filtering by exact timestamp is useless for the perf team. By typing `2018-02-28` they expect to all the items which belong to period of 00:00:000 to 23:59:59. Currently mongodb converts `2018-02-28` to `2018-02-28 00:00:00` which doesn't match anything at all.